bundles/org.openhab.binding.mideaac/src/main/resources/OH-INF/addon/addon.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<addon:addon id="mideaac" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:addon="https://openhab.org/schemas/addon/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/addon/v1.0.0 https://openhab.org/schemas/addon-1.0.0.xsd">
+
+	<type>binding</type>
+	<name>MideaAC Binding</name>
+	<description>This is the binding for MideaAC.</description>
+	<connection>local</connection>
+
```

Comment:
Do you know if the ac advertises itself by upnp or ssdp or other protocol? If so the binding can be suggested during setup.

Reply:
I can look at this next but would like to have as an enhancement.  I'll need to study how the suggestions work.  The HA python website I referenced in the README has a discover option, but I'm not sure the device is constantly advertising or just responds to the discovery packet.  Note logs
```
2024-10-16 15:27:20.355 [DEBUG] [al.discovery.MideaACDiscoveryService] - Start scan for Midea AC devices.
2024-10-16 15:27:20.357 [TRACE] [al.discovery.MideaACDiscoveryService] - Discovering: 255.255.255.255
2024-10-16 15:27:20.361 [TRACE] [al.discovery.MideaACDiscoveryService] - Broadcast discovery package sent to port: 6445
2024-10-16 15:27:20.363 [TRACE] [al.discovery.MideaACDiscoveryService] - Broadcast discovery package sent to port: 20086
2024-10-16 15:27:20.407 [DEBUG] [al.discovery.MideaACDiscoveryService] - Midea AC device discovery returned package with length 208
```